File: trunk/Projects/Framework-Samples/NHibernate/CSharp/Sample.Data/ManagerObjects/ProductManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using Sample.Data.Generated.BusinessObjects;
using Sample.Data.Generated.Base;

namespace Sample.Data.Generated.ManagerObjects
{
    public partial interface IProductManager : IManagerBase<Product, string>
    {
	}
	
	partial class ProductManager : ManagerBase<Product, string>, IProductManager
    {
	}
}

File: Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/ProductManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface IProductManager
	'	Inherits IManagerBase(Of Product, System.String)
	'End Interface

	Partial Class ProductManager
		Inherits ManagerBase(Of Product, System.String)
		Implements IProductManager
	End Class
End Namespace
